test/integration/registration/integration_suite_test.go

```diff
@@ -216,6 +216,10 @@ var _ = ginkgo.BeforeSuite(func() {
 		go func() {
 			defer ginkgo.GinkgoRecover()
 			m.ImportOption.APIServerURL = cfg.Host
+			m.EnabledRegistrationDrivers = []string{helpers.CSRAuthType, helpers.AwsIrsaAuthType}
```

Comment:
I remember I actually remove this part on purpose, so we do not trigger aws driver in the other integration test to avoid error messages.